scripts/globals/mobskills/hiemal_storm.lua

```diff
@@ -0,0 +1,30 @@
+---------------------------------------------
+--  Hiemal Storm
+--
+--  Description: Extreme directional AoE ice damage for 200-1400 points
+--  Type: Breath
+--  Utsusemi/Blink absorb: Ignores shadows
+--  Range: Unknown cone
+--  Notes: Used only by Snoll Tzar
+---------------------------------------------
+require("scripts/settings/main")
```

Comment:
It doesn't look like any settings are used in this file, so this require isn't needed.